tests/modules/hub.py

```diff
@@ -4,9 +4,30 @@
 import torchvision.models as models
 import timm
 from transformers import BertModel, BertTokenizer, BertConfig
+import os
+import sys
 
 torch.hub._validate_not_a_forked_repo = lambda a, b, c: True
 
+torch_version = torch.__version__
+snapshot_file = 'model_snapshot.txt'
+skip_download = False
+
+# If model repository already setup
```

Comment:
I feel like what this should be is a list of models which have been downloaded and we check if the model we are downloading is in the list, because we can always add more. And this would require someone to delete the file to redownload

Reply:
Updated the code.